UnityProject/Assets/Scripts/Messages/Server/ShootMessage.cs

```diff
@@ -115,7 +115,7 @@ public class CastProjectileMessage : ServerMessage
 		/// <summary>
 		/// Projectile being shot
 		/// </summary>
-		public Guid ProjectilePrefab;
+		public string ProjectilePrefabName;
```

Comment:
I feel like this is a more flimsy reference

Reply:
its the same way the guns shoot their projectiles

Reply:
ok cool